Dockerfile

```diff
@@ -15,6 +15,13 @@ ENV PATH $NVM_DIR/versions/node/v$NODE_VERSION/bin:$PATH
 # Add bin directory used by `pip install --user`
 ENV PATH "/home/docker/.local/bin:${PATH}"
 
+# Infrastructure tools
+# gettext is used for django to compile .po to .mo files.
+RUN apt-get update
+RUN apt-get install apt-utils -y
+RUN apt-get update
+RUN apt-get install gettext python3-pip -y
```

Comment:
有什麼特別的緣故需要執行第二次 update 嗎? (好奇)... 想說是否可以合併成一行
```Docker
RUN apt-get install apt-utils gettext python3-pip -y
```
還是說裝完 apt-utils 需要特別 update 一次才會生效 (?)

Reply:
如果沒有先裝 apt-utils 會有如下 warning

```
Get:1 http://deb.debian.org/debian buster/main amd64 libapt-inst2.0 amd64 1.8.2 [204 kB]
Get:2 http://deb.debian.org/debian buster/main amd64 apt-utils amd64 1.8.2 [421 kB] 
debconf: delaying package configuration, since apt-utils is not installed               
Fetched 625 kB in 0s (5345 kB/s)                                                        
```
在我們的使用情境裡（裝 gettext 等以及後面從 source build 一些工具如 nvm）這個 debconf 的警告「似乎」不至於影響到，但以防萬一就還是分開處理盡量讓流程在我預期的環境下工作。